models/project/project.go

```diff
@@ -61,6 +62,89 @@ func FindProject(projectID int64) *ProjectStruct {
 	}
 }
 
+func Create(userId int64, title string, description string, repositoryID int64, repositoryOwner string, repositoryName string, oauthToken sql.NullString) (p *ProjectStruct, e error) {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 13 of function Create() is high (> 10)
* [golint] exported function Create should have comment or be unexported
* [golint] func parameter userId should be userID